Develop

davypaperboy opened the pull request.

Seneral commented:
What is this for? Please provide a description:)
Looks like a merge from develop to master, which currently is not applicable because develop has not been revisited and accepted to be fully stable...

davypaperboy commented:
Sorry, was just checking out your code & was trying to switch to develop 
branch in an attempt to get one of the example working. I've just 
switched to windows, I'd never used the github GUI before and it created 
a merge request. Back to the command line, much safer!

I've removed the fork, sorry about that!

Davy


On 27/01/2017 19:54, Levin G. wrote:
>
> What is this for? Please provide a description:)
> Looks like a merge from develop to master, which currently is not 
> applicable because develop has not been revisited and accepted to be 
> fully stable...
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub 
> <https://github.com/Baste-RainGames/Node_Editor/pull/119#issuecomment-275758746>, 
> or mute the thread 
> <https://github.com/notifications/unsubscribe-auth/ABIw13At6mAmBDo4BRSn98WiMjen4LKdks5rWkuJgaJpZM4Lvrpi>.
>

Seneral commented:
Ah ok no problem:)
